### PR TITLE
Release v2.9

### PR DESCRIPTION
**Pull request by rishi-104276**
## Description
This PR contains changes we are pushing out for our 2.9 release.

## Jira
https://uipath.atlassian.net/browse/TS-1756

## Testing
Tested over the preview build 
creating entirely new service connection for paas deployments.
checked for existing external app service connections.

For both scenarios was able to pack and deploy to the respective folders.

## Screenshots
image
image
image
image

### CODE REVIEW

File: src/main/java/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
  */
 public class ExternalAppAuthenticationEntry extends SelectEntry {
 
+	private final String identityUrl;
```
Comment (ganeshborle) [inline src/main/java/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry.java:28]: indentation

File: src/main/java/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry.java [hunk 2]
```diff
@@ -76,8 +78,16 @@ public String getApplicationSecret() {
...
+    /**
+     * Gets the identityUrl
+     * @return String identityUrl
+     */
+    public String getIdentityUrl() {
+		return identityUrl;
+	}
 
-    @Override
+	@Override
```
Comment (ganeshborle) [inline src/main/java/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry.java:90]: indentation

File: src/main/java/com/uipath/uipathpackage/models/AuthenticatedOptions.java [hunk 3]
```diff
@@ -113,4 +114,13 @@ public TraceLevel getTraceLevel() {
...
     }
+
+	public String getIdentityUrl() {
+		return identityUrl;
+	}
+
+	public void setIdentityUrl(String identityUrl) {
+		this.identityUrl = identityUrl;
+	}
+
```
Comment (ganeshborle) [inline src/main/java/com/uipath/uipathpackage/models/AuthenticatedOptions.java:125]: indentation

File: src/main/resources/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry/config.jelly [hunk 4]
```diff
@@ -1,5 +1,8 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+	<f:entry field="identityUrl" title="${%IdentityUrl}">
+        <f:textbox />
+    </f:entry>
```
Comment (ganeshborle) [inline src/main/resources/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry/config.jelly:5]: indentation

**Review by ganeshborle — CHANGES_REQUESTED** [review]

File: src/main/resources/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry/config.jelly (hunk 4 above)
Comment (rishi-104276) [reply src/main/resources/com/uipath/uipathpackage/entries/authentication/ExternalAppAuthenticationEntry/config.jelly:5]: done

File: src/main/java/com/uipath/uipathpackage/models/AuthenticatedOptions.java (hunk 3 above)
Comment (rishi-104276) [reply src/main/java/com/uipath/uipathpackage/models/AuthenticatedOptions.java:125]: done

**Review by ganeshborle — APPROVED** [review]
LGTM

**Pull request merged**